howto/port/install-as-system-app.md

```diff
@@ -0,0 +1,70 @@
+Anbox Cloud allows user applications to be installed as Android system apps. This can be done by using the `aam install-system-app` command.
```

Comment:
Can you add a short explanation on the difference between a system app and a normal app? Why would a user want an app to be installed as system app?

Reply:
Will do

Reply:
Will do